types/final-form-focus/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Do not add rules to tslint and disable the rule for package.. Fix errors instead or disable error at the location it fails.

Reply:
Ok sure thing 😊
Just pushed the changes, let me know what you think, @sheetalkamat